test/datasets/transformations.jl

```diff
@@ -33,4 +33,18 @@ include("../imports.jl")
         jdata = joinobs(data1, data2)
         @test getobs(jdata, 15) == 15
     end
+
+    @testset ExtendedTestSet "tokenize" begin
+        data1 = "The quick rabbit jumps over the lazy fox."
+        data2 = " The quick rabbit jumps over the lazy fox."
+        tdata1 = Datasets.tokenize(:words,data1)
+        tdata2 = Datasets.tokenize(:words,data2)
+        tdata3 = Datasets.tokenize(:chars,data1)
+        tdata4 = Datasets.tokenize(:chars,data2)
+        @test getobs(tdata1,3) == "rabbit"
+        @test getobs(tdata2,3) == "rabbit"
+        @test getobs(tdata3,3) == "e"
+        @test getobs(tdata4,3) == "h"
+        #TODO: add stream tests
```

Comment:
By "stream tests", do you mean enumerating more than one token? If so, I agree those should be added.

Reply:
No, I meant to take a stream as input, which I have yet to figure out how to create as a test. By enumerate more than one token, do you mean check more than one index?